.travis.yml

```diff
@@ -90,19 +87,19 @@ jobs:
       services:
         - docker
       env:
-        # GITHUB_ACCESS_TOKEN
-        - secure: "iH5SDg88SsrTRMgX+a9qMZVvfyzhZhipsnuF927oG5BQh0unXUIzqgN11mHLlea/tyCkyz5jfZfz5kpnK+GKRZBd0RmozqiyB7XptTp+O9Oq+rXhwN4Xml+Q9Vw2ndJ+70TmnaJp1BKq8y9Bw9eI3YWPuOQl0VZgElZSgWdRltpgntPrO3piTd4EHky+RCcp/9l877Il0L+hdrC+pUn8n/qrTvODaPXc86AmjsgWoYburej9+xTLLa5tG8MPTF9QqvxHWmz1alDABxw2ROgySXgPcp+iKMWrkTGY+kCNxregg2m6F8sqnmQJqxJ7rjU8NNBfGLf/vtfU1ocAnWW82ej+ewFOlynvDLnIaBh0CyXcqe8B7XBmwufPk3Ycu7wUERuJHa+tgjiwDzHNogCWuq6r6a5WGnmuDdeu9EY0IU1QnVIGdKr+aZ5TnkMOi9FJbjc/qh/8ZJpaVZhmC/cGau4nZ1q9KC1RznH6WXjG43CEVf1QxTiYIZuuH+J1qhESiykVwUOkdjHm3GjKYxkW2yK4az0DV4LgQmwsBAhCunBZHcTUwy6lhl5bW39mNiaYshg7vFdYc7mmqiK/VH5WLmUN3UOA9QmsSV6t7y5L/3RRbFNDBJF0NXxbNDRTQm6PlsWXkCKRtIptqR2B8hPMMHnS5pTOv7zKH1j0JDKRcEo="
+        # DOCKERHUB_ACCESS_TOKEN
+        - secure: "cLVBcS5xFdhPGFH17xI5Au+Ui3LpcgDjJT/XTC5sGFf/UKkhB1UfyodxXvzxLuRYR3fAaoETAV8gB5wZ/nwVsJ4oAWzu2wvrBp/ORvwF77zyIro/S4ZnTev3gDkaU8We52MG+Yc3mUAXClz/pock7d5RKMYBony8mHB29iHohe7DNFvNHogMiLgjajX8DHUIPwroBsL3DGSFIFaCkw+dkv5vSRWa69vCMpQl0uDaB+2vm+mKIs8RA01eePOL+tlyLnkwhM7zf8Km+Bqkh5SbHNopnREFAcVdX6OU9dfGb9iY6ZasVU/tQbNVHeUPQTirhL6ICYLPWuCiKqBEHMLDj7WgbXoUvqCgE95723RGff5XUjsR/SgCntB/MEj48adylKDdibm4qV6iqtKvT4NpXCbQLORg+bRtgG+8klP/ZMYTYUTnMHikO+kP6Dz3Lm8QVDSeCn+WdseeQMIsfkjBwdkKi51l5nXP/ObktjiqFiT51UX3ieER88JtAZKgd2Hbc4LrjvvrMHLWiNTG/BghVQl/1/ls/3p5BXF8O/VJO++3f6bZ1kSlAZ4ik0uxoSajEHt1vkVU2S8OdTOPvVv/XG40RZfnVTX0mOw+2VsLLJ03smoWw2wkPl1T/DvrCjvRVnHDD3t3AX/0z2zIJ2jkaTgxC/wi5Ocp2U7BA2usW74="
       workspaces:
         use:
           - build-docker
       script:
-        - echo "$GITHUB_ACCESS_TOKEN" | docker login docker.pkg.github.com --username juspay --password-stdin
+        - echo "$DOCKERHUB_ACCESS_TOKEN" | docker login --username juspayin --password-stdin
         - docker load -i fencer.tar.gz
         # Tag the image with commit hash and push
         - HASH=$(git log -1 --pretty=%h)
-        - docker tag juspay/fencer "docker.pkg.github.com/juspay/fencer/fencer:$HASH"
-        - docker push "docker.pkg.github.com/juspay/fencer/fencer:$HASH"
+        - docker tag juspayin/fencer "juspayin/fencer:$HASH"
+        - docker push "juspayin/fencer:$HASH"
         # Tag the same image with "master" and push
-        - docker tag juspay/fencer docker.pkg.github.com/juspay/fencer/fencer:master
-        - docker push docker.pkg.github.com/juspay/fencer/fencer:master
-      if: branch = master AND type = push
```

Comment:
why is the check `branch = master` not needed anymore?

Reply:
It's a part of the `TEST COMMIT WILL REVERT LATER` commit. I'll revert it once it becomes evident that the PR works.